4.8/Dockerfile

```diff
@@ -6,15 +6,19 @@ RUN groupadd --gid 1000 node \
 # gpg keys listed at https://github.com/nodejs/node
 RUN set -ex \
   && for key in \
-    9554F04D7259F04124DE6B476D5A82AC7E37093B \
+    # Current keys from https://github.com/nodejs/node#release-team
```

Comment:
We already node the source URL on line 6 so this is redundant. However, I think updating the current URL to with the full link to the release team section (https://github.com/nodejs/node#release-team) would be good